prescriptions/_predictable_stacks/ps_cv_torch.yaml

```diff
@@ -0,0 +1,35 @@
+units:
+  wraps:
+  - name: ThothPSCVTorchBaseImageWrap
+    type: wrap
+    should_include:
+      adviser_pipeline: true
+      runtime_environments:
+        base_images: [null]  # Only if no base image is used.
+    match:
+    - state:
+        resolved_dependencies:
+        - name: jupyter-tensorboard
+    - state:
+        resolved_dependencies:
+        - name: opencv-python
+    - state:
+        resolved_dependencies:
+        - name: pillow
+          version: ">8.3.0"
+    - state:
+        resolved_dependencies:
+        - name: tensorboard
+    - state:
+        resolved_dependencies:
+        - name: torch
+    - state:
+        resolved_dependencies:
+        - name: torchvision
+    run:
+      advised_manifest_changes:
+      - base_images: "quay.io/thoth-station/ps-cv-pytorch:v0.1.2"
```

Comment:
Please remove the `advised_manifest_changes` part until the implementation is available in thamos. It might look different.